system_tests_async/conftest.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2016 Google LLC
```

Comment:
I don't think you should need a whole separate async system test directory and conftest. Just add it to the existing system tests dir and put any new fixtures you are adding in the existing system_tests/conftest.py . You may have to rename some fixtures to get this to work.

Reply:
Will look into doing this refactoring in the 3rd PR